packages/builders/src/interactions/commands/chatInput/ChatInputCommandSubcommands.ts

```diff
@@ -0,0 +1,103 @@
+import type { JSONEncodable } from '@discordjs/util';
+import type {
+	APIApplicationCommandSubcommandOption,
+	APIApplicationCommandSubcommandGroupOption,
+} from 'discord-api-types/v10';
+import { ApplicationCommandOptionType } from 'discord-api-types/v10';
+import { Mixin } from 'ts-mixer';
+import { isValidationEnabled } from '../../../util/validation.js';
+import { chatInputCommandSubcommandGroupPredicate, chatInputCommandSubcommandPredicate } from './Assertions.js';
+import { SharedChatInputCommandOptions } from './mixins/SharedChatInputCommandOptions.js';
+import type { SharedNameAndDescriptionData } from './mixins/SharedNameAndDescription.js';
+import { SharedNameAndDescription } from './mixins/SharedNameAndDescription.js';
+
+export interface ChatInputCommandSubcommandGroupData {
+	options?: ChatInputCommandSubcommandBuilder[];
+}
+
+/**
+ * Represents a folder for subcommands.
+ *
+ * @see {@link https://discord.com/developers/docs/interactions/application-commands#subcommands-and-subcommand-groups}
+ */
+export class ChatInputCommandSubcommandGroupBuilder
+	extends SharedNameAndDescription
+	implements JSONEncodable<APIApplicationCommandSubcommandGroupOption>
+{
+	protected declare readonly data: ChatInputCommandSubcommandGroupData & SharedNameAndDescriptionData;
+
+	/**
+	 * Adds a new subcommand to this group.
+	 *
+	 * @param input - A function that returns a subcommand builder or an already built builder
+	 */
+	public addSubcommand(
```

Comment:
Ngl... I never liked the whole add multiple at once, especially since its something super easy to do at the user code level (`subcommands.forEach(item => builder.addSubcommand(item))`). That said 🤷‍♂️ I'd rather we kept it single